.github/workflows/ci.yml

```diff
@@ -11,18 +11,22 @@ jobs:
       fail-fast: false
       matrix:
         distro:
-        - centos-8
         - centos-9
         - fedora-35
         - fedora-36
+        include:
+          - distro: centos-8
+            env: py36
+          - distro: centos-8
+            env: py38
     container:
       image: quay.io/ovirt/imageio-test-${{matrix.distro}}
       # Required to create loop devices.
       options: --privileged
     steps:
     - uses: actions/checkout@v2
     - name: Run tests
-      run: ci/test.sh
+      run: ci/test.sh ${{matrix.env}}
```

Comment:
So matix.env si empty string when not specified?

This means we need to handle missing env in the script which will make it more complicated.